lib/RESTClient.js

```diff
@@ -412,8 +417,12 @@ class RESTClient {
                 ret.push(data);
                 retLen += data.length;
             }).on('error', callback).on('end', () => {
-                this.endRespond(res, Buffer.concat(ret, retLen).toString(),
-                    log, callback);
+                if (req.url === '/_/healthcheck') {
+                    callback(null, res);
```

Comment:
if you can't its not a big deal :) if the only thing you need is the response object, you can override the callback send to endResponse and override the return data :)